src/main/java/org/zalando/fahrschein/domain/Cursor.java

```diff
@@ -1,6 +1,6 @@
 package org.zalando.fahrschein.domain;
 
-public final class Cursor {
+public class Cursor {
```

Comment:
Can you give an example, where you want to mock a Cursor? It has only 2 `String` fields.